### PR TITLE
Create C06.csv

### PR DESCRIPTION
**Pull request by amilan17**
add C06 file for units

### CODE REVIEW

File: C06.csv [hunk 1]
```diff
@@ -0,0 +1,161 @@
+CodeFigure,UnitType,Meaning,conventional,IA5-ASCII,ITA2,SIDefinition,Note,NoteID,Status
+001,Base SI units,metre,m,m,M,,(see Note 1),19,Operational
```
Comment (david-i-berry) [inline C06.csv:2]: Note 1 is applicable to 001 - 082, plus the SI prefixes.

File: C06.csv [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+042,Derived SI units with special names,henry,H,H,H,kg m^2 s^-2 A^-2,,,Operational
+060,Derived SI units with special names,degree Celsius,˚C,Cel,CEL,K+273.15,,,Operational
+070,Derived SI units with special names,lumen,lm,lm,LM,cd sr,,,Operational
+071,Derived SI units with special names,lux,lx,lx,LX,cd sr m^-2,,,Operational
+080,Derived SI units with special names,becquerel,Bq,Bq,BQ,s^-1,,,Operational
+081,Derived SI units with special names,grey,Gy,Gy,GY,m^2 s^-2,,,Operational
+082,Derived SI units with special names,sievert,Sv,Sv,SV,m^2 s^-2,,,Operational
+110,"Other, non-SI, units recognized by CGPM",degree (angle),˚,deg,DEG,,(see Note 4),10,Operational
+111,"Other, non-SI, units recognized by CGPM",minute (angle),',',MNT,,,,Operational
+112,"Other, non-SI, units recognized by CGPM",second (angle),'','',SEC,,,,Operational
```
Comment (david-i-berry) [inline C06.csv:32]: two ' or a double quote", e.g.

`112,"Other, non-SI, units recognized by CGPM",second (angle),"""","""",SEC,,,,Operational`

File: C06.csv [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+160,"Other, non-SI, units recognized by CGPM",electron volt,eV,eV,EV,,,,Operational
+161,"Other, non-SI, units recognized by CGPM",atomic mass unit,u,u,U,,,,Operational
+170,"Other, non-SI, units recognized by CGPM",astronomic unit,AU,AU,ASU,,,,Operational
+171,"Other, non-SI, units recognized by CGPM",parsec,pc,pc,PRS,,,,Operational
+200,Non-SI Units tolerated because of widespread use,nautical mile, ,,,,(see Note 7),21,Operational
+201,Non-SI Units tolerated because of widespread use,knot,kt,kt,KT,,,,Operational
+210,Non-SI Units tolerated because of widespread use,decibel,dB,dB,DB,,(see Note 6),17,Operational
+220,Non-SI Units tolerated because of widespread use,hectare,ha,ha,HAR,,,,Operational
+230,Non-SI Units tolerated because of widespread use,week, ,,,,,,Operational
+231,Non-SI Units tolerated because of widespread use,year,a,a,A,,,,Operational
```
Comment (david-i-berry) [inline C06.csv:47]: Class with prefix for atto

File: C06.csv [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+300,Other Units as used by WMO,per cent,%,%,PERCENT,,,,Operational
```
Comment (david-i-berry) [inline C06.csv:48]: Note 7?

File: C06.csv [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+301,Other Units as used by WMO,parts per thousand,‰,0/00,PERTHOU,,,,Operational
+310,Other Units as used by WMO,eighths of cloud,okta,okta,OKTA,,,,Operational
+320,Other Units as used by WMO,degrees true,˚,deg,DEG,,,,Operational
+321,Other Units as used by WMO,degrees per second,degree/s,deg/s,DEG/S,,,,Operational
+350,Other Units as used by WMO,degrees Celsius,˚ C,C,C,,(see Note 8),13,Operational
```
Comment (david-i-berry) [inline C06.csv:53]: Delete the space between the degree symbol and the C, they are one unit. With the space we have degree Coulomb.

Comment (david-i-berry) [reply C06.csv:53]: this is also applicable to other entries below.

File: C06.csv [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+522,Other Units as used by WMO,centibars per 12 hours,cb/12 h,cb/12 h,CB/12 HR,,,,Operational
+523,Other Units as used by WMO,dekapascal,daPa,daPa,DAPAL,,,,Operational
+530,Other Units as used by WMO,hectopascal,hPa,hPa,HPAL,,,,Operational
+531,Other Units as used by WMO,hectopascals per second,hPa s^-1,hPa/s,HPAL/S,,,,Operational
+532,Other Units as used by WMO,hectopascals per hour,hPa h^-1,hPa/h,HPAL/HR,,,,Operational
+533,Other Units as used by WMO,hectopascals per 3 hours,hPa/3 h,hPa/3 h,HPAL/3 HR,,,,Operational
+535,Other Units as used by WMO,nanobar = hPa 10^-6,nbar,nbar,NBAR,,,,Operational
+620,Other Units as used by WMO,grams per kilogram,g kg^-1,g/kg,G/KG,,,,Operational
+621,Other Units as used by WMO,grams per kilogram per second,g kg^-1 s^-1,g kg-1 s-1,,,,,Operational
+622,Other Units as used by WMO,kilograms per kilogram,kg kg^-1,kg/kg,,,,,Operational
```
Comment (david-i-berry) [inline C06.csv:74]: Missing ITA2 abbreviation.

File: C06.csv [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+623,Other Units as used by WMO,kilograms per kilogram per second,kg kg^-1 s^-1,kg kg-1 s-1,,,,,Operational
+624,Other Units as used by WMO,kilograms per square metre,kg m^-2,kg m-2,,,,,Operational
+630,Other Units as used by WMO,acceleration due to gravity,g,g,,,,,Operational
+631,Other Units as used by WMO,geopotential metre,gpm,gpm,,,,,Operational
+710,Other Units as used by WMO,millimetre,mm,mm,MM,,,,Operational
+711,Other Units as used by WMO,millimetres per seconds,mm s^-1,mm/s,MM/S,,,,Operational
+712,Other Units as used by WMO,millimetres per hour,mm h^-1,mm/h,MM/HR,,,,Operational
+713,Other Units as used by WMO,millimetres per the sixth power per cubic metre,mm^6 m^-3,mm6 m-3,,,,,Operational
```
Comment (david-i-berry) [inline C06.csv:82]: millimetres `to` the sixth power not millimetres `per` the sixth power

File: C06.csv [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+815,Other Units as used by WMO,watts per cubic metre per steradian,W m^-3 sr^-1,W m-3 sr-1,,,,,Operational
+820,Other Units as used by WMO,siemens per metre,S m^-1,S/m,,,,,Operational
+825,Other Units as used by WMO,square degrees,degrees^2,deg^2,,,,,Operational
+830,Other Units as used by WMO,becquerel seconds per cubic metre,Bq s m^-3,Bq s m-3,,,,,Operational
+835,Other Units as used by WMO,decibels per metre,dB m^-1,dB/m,,,,,Operational
+836,Other Units as used by WMO,decibels per degree,dB degree^-1,dB/deg,,,,,Operational
+841,Other Units as used by WMO,pH unit,pH unit,pH unit,,,,,Operational
+842,Other Units as used by WMO,N units,N units,N units,,,,,Operational
+843,Other Units as used by WMO,Nephelometric turbidity units,NTU,,,,,,Operational
+844,Other Units as used by WMO,Total electron content unit,TECU,,,10^16 Electrons m^�2,,,
```
Comment (david-i-berry) [inline C06.csv:141]: m^`-`2, elongated dash has been used, change to regular.

**Review by david-i-berry — APPROVED** [review]

File: C06.csv [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+805,Other Units as used by WMO,joules per square metre,J m^-2,J m-2,,,,,Operational
+806,Other Units as used by WMO,joules per kilogram,J kg^-1,J/kg,,,,,Operational
+810,Other Units as used by WMO,watts per metre per steradian,W m^-1 sr^-1,W m-1 sr-1,,,,,Operational
+811,Other Units as used by WMO,watts per square metre,W m^-2,W m-2,,,,,Operational
+812,Other Units as used by WMO,watts per square metre per steradian,W m^-2 sr^-1,W m-2 sr-1,,,,,Operational
+813,Other Units as used by WMO,watts per square metre per steradian centimetre,W m^-2 sr^-1 cm,W m-2 sr-1 cm,,,,,Operational
+814,Other Units as used by WMO,watts per square metre per steradian metre,W m^-2 sr^-1 m,W m-2 sr-1 m,,,,,Operational
+815,Other Units as used by WMO,watts per cubic metre per steradian,W m^-3 sr^-1,W m-3 sr-1,,,,,Operational
+820,Other Units as used by WMO,siemens per metre,S m^-1,S/m,,,,,Operational
+825,Other Units as used by WMO,square degrees,degrees^2,deg^2,,,,,Operational
```
Comment (amilan17) [inline C06.csv:134]:
```suggestion
825,Other Units as used by WMO,square degrees,degrees^2,deg2,,,,,Operational
```

**Pull request merged**